website/pages/explore/dir-content/pwa.asciidoc

```diff
@@ -1,17 +0,0 @@
-= PWA FILE
-
-[.directory]
-== PWA
-
-Progresive web applications (PWAs) are web application that offer better user experience than the traditional ones. In general, they solve problems related with reliability and speed:
-
-* Reliability: PWAs are stable. In this context stability means than even with slow connections or even with no network at all, the application still works. To achieve this, some basic resources like styles, fonts, requests, …​ are stored; due to this caching, it is not possible to assure that the content is always up-to-date.
-
-* Speed: When an users opens an application, he or she will expect it to load almost inmediately (almost 53% of users abandon sites that take longer that 3 seconds, source: https://developers.google.com/web/progressive-web-apps/#fast).
-
-[.common-links]
-=== Links
-
-<</website/pages/docs/master-devon4ng.asciidoc_angular.html#guide-angular-pwa.asciidoc, Angular PWA>>
-
-<</website/pages/docs/master-devon4ng.asciidoc_ionic.html#guide-ionic-pwa.asciidoc, Ionic 4 PWA>>
```

Comment:
could we restore this file? It's really good
Progressive Web Apps are a technology / paradigm for mobile development. So maybe we can add it into a mobile development section?